dependencies/vaticle/repositories.bzl

```diff
@@ -36,7 +36,7 @@ def vaticle_typedb_protocol():
         name = "vaticle_typedb_protocol",
         remote = "https://github.com/vaticle/typedb-protocol",
         # NOTE: the sync-marker is also used for workspace status by Bazel!
-        commit = "aea498d852d763c20d9ecc7d11423d7506964b95",  # sync-marker: do not remove this comment, this is used for sync-dependencies by @vaticle_typedb_protocol
+        tag = "2.28.4",  # sync-marker: do not remove this comment, this is used for sync-dependencies by @vaticle_typedb_protocol
```

Comment:
I decided to highlight this "downgrade" of the protocol version here before the 2.28.4 release of the driver. Krishnan updated dependencies too early and it has been propagated everywhere now (causing the propagation of the updated by these dependencies protocol as well). I'll put this protocol commit here after the release.